lldb/source/Target/StackFrame.cpp

```diff
@@ -1397,7 +1398,9 @@ ValueObjectSP GetValueForOffset(StackFrame &frame, ValueObjectSP &parent,
     return parent;
   }
 
-  for (int ci = 0, ce = parent->GetNumChildren(); ci != ce; ++ci) {
+  for (int ci = 0, ce = llvm::expectedToStdOptional(parent->GetNumChildren())
```

Comment:
I'm afraid that calls to `expectedToStdOptional` will just silence important warnings.